statsmodels/tsa/tests/test_arima.py

```diff
@@ -2301,6 +2301,8 @@ def test_arima_pickle():
     assert_almost_equal(res.fittedvalues, pkl_res.fittedvalues)
     assert_almost_equal(res.pvalues, pkl_res.pvalues)
 
+def test_arima_not_implemented():
+    assert_raises(NotImplementedError, ARIMA.from_formula,' WUE ~ 1 + SFO3 ',5)
```

Comment:
space after comma, pep-8
using valid data would be "nicer" but not worth the extra code, given the the exception is specified.